<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QMainWindow" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chess Game - Главное меню</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fefefe;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="leftMargin">
     <number>40</number>
    </property>
    <property name="topMargin">
     <number>30</number>
    </property>
    <property name="rightMargin">
     <number>40</number>
    </property>
    <property name="bottomMargin">
     <number>30</number>
    </property>
    <property name="spacing">
     <number>20</number>
    </property>

    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>Шахматы</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="font">
       <font>
        <pointsize>26</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #1e3a5f; background-color: #dbeafe; padding: 18px; border-radius: 18px; border: 1px solid #93c5fd;</string>
      </property>
     </widget>
    </item>

    <item>
     <layout class="QVBoxLayout" name="playersLayout">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <widget class="QLineEdit" name="whitePlayerEdit">
        <property name="placeholderText">
         <string>Имя игрока за белых</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #111827; background-color: white; border: 1px solid #d1d5db; padding: 8px; border-radius: 8px; font-size: 14px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="blackPlayerEdit">
        <property name="placeholderText">
         <string>Имя игрока за чёрных</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #111827; background-color: white; border: 1px solid #d1d5db; padding: 8px; border-radius: 8px; font-size: 14px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>

    <item>
     <layout class="QVBoxLayout" name="buttonsLayout">
      <property name="spacing">
       <number>12</number>
      </property>

      <item>
       <widget class="QPushButton" name="startGameButton">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>44</height>
         </size>
        </property>
        <property name="text">
         <string>Начать игру</string>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton {
    color: white;
    background-color: #3b82f6;
    border: 1px solid #2563eb;
    border-radius: 14px;
    padding: 10px;
    font-weight: bold;
    font-size: 16px;
}
QPushButton:hover {
    background-color: #60a5fa;
}
QPushButton:pressed {
    background-color: #2563eb;
}
         </string>
        </property>
       </widget>
      </item>

      <item>
       <widget class="QPushButton" name="settingsButton">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>44</height>
         </size>
        </property>
        <property name="text">
         <string>Настройки</string>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton {
    color: white;
    background-color: #60a5fa;
    border: 1px solid #3b82f6;
    border-radius: 14px;
    padding: 10px;
    font-weight: bold;
    font-size: 16px;
}
QPushButton:hover {
    background-color: #93c5fd;
}
QPushButton:pressed {
    background-color: #3b82f6;
}
         </string>
        </property>
       </widget>
      </item>

      <item>
       <widget class="QPushButton" name="statsButton">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>44</height>
         </size>
        </property>
        <property name="text">
         <string>Статистика</string>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton {
    color: white;
    background-color: #34d399;
    border: 1px solid #10b981;
    border-radius: 14px;
    padding: 10px;
    font-weight: bold;
    font-size: 16px;
}
QPushButton:hover {
    background-color: #6ee7b7;
}
QPushButton:pressed {
    background-color: #10b981;
}
         </string>
        </property>
       </widget>
      </item>

      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>44</height>
         </size>
        </property>
        <property name="text">
         <string>Выход</string>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton {
    color: white;
    background-color: #ef4444;
    border: 1px solid #dc2626;
    border-radius: 14px;
    padding: 10px;
    font-weight: bold;
    font-size: 16px;
}
QPushButton:hover {
    background-color: #f87171;
}
QPushButton:pressed {
    background-color: #dc2626;
}
         </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>

   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
